versioned/storage/cache/src/main/java/org/projectnessie/versioned/storage/cache/CachingPersistImpl.java

```diff
@@ -126,7 +126,7 @@ public Obj[] fetchObjs(@Nonnull ObjId[] ids) throws ObjNotFoundException {
       Obj o = backendResult[i];
       if (o != null) {
         r[i] = o;
-        cache.put(o);
+        cache.putLocal(o);
```

Comment:
Should we use `putLocal()` in `fetchObj()` (singular) too?

Reply:
yes